spec/acceptance/class_spec.rb

```diff
@@ -48,7 +48,7 @@ class { 'grafana':
             {
               name            => 'default',
               orgId           => 1,
-              fiolder         => '',
```

Comment:
how did that ever work?

Reply:
Not sure, but I spent an embarrassing amount of time trying to figure out how to put provisioned dashboards in Grafana folders because of the typos.